EIPS/eip-4672.md

```diff
@@ -0,0 +1,56 @@
+---
+eip: 4672
+title: NONCE opcode
+description: A new EVM opcode to get the current nonce of an address.
+author: 0xfoobar (@0xfoobar)
+discussions-to: https://ethereum-magicians.org/t/eip-4672-account-nonce-opcode/8171
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-02-01
+---
+
+## Abstract
+
+We propose an additional EVM opcode, `NONCE` (0x47), which could provide novel protection against sybil attacks and expand the smart contract design space. This would return `0` for an unused externally-owned address (EOA), `0` for a smart contract address, and, for example, `2` for an EOA which has sent 3 transactions. The account.nonce opcode would exactly track the nonce of the last transaction originating from that address.
```

Comment:
Hm, I think it might be best to leave out the examples of nonces in the abstract.

Taking a peek at the [execution-specs](https://ethereum.github.io/execution-specs/autoapi/ethereum/spurious_dragon/vm/interpreter/index.html#process-create-message), for example, it looks like a newly created smart contract has a nonce of 1!

There's also still a reference to `account.nonce opcode`.